chia/cmds/wallet_funcs.py

```diff
@@ -97,8 +82,8 @@ async def get_transaction(args: dict, wallet_client: WalletRpcClient, fingerprin
         print(e.args[0])
         return
 
-    print_transaction(
-        tx,
+    print_coins_transactions(
```

Comment:
I'd would have probalby not changed this function as it's orthogonal to the functionality you are adding.

Reply:
oops i see what i did

Reply:
ignore the comment that i deleted, that was unrelated

Reply:
I guess I don't understand because I see print_coins_transactions added to show.py and it literally does `if (a) do a else do b` so afaict your duped it anyway

Reply:
> I guess I don't understand because I see print_coins_transactions added to show.py and it literally does `if (a) do a else do b` so afaict your duped it anyway

no wait, fair point

Reply:
fixed it, thank you for that.